include/swift/SIL/LinearLifetimeChecker.h

```diff
@@ -74,22 +74,10 @@ class LinearLifetimeChecker {
   /// Returns true if we completed the consuming use set and discovered that \p
   /// consumingUse is not strongly control equivalent to value (meaning
   /// consumingUse is not in the same loop in the loop nest as value).
```

Comment:
It's not part of this PR, but I have to say: convention is to return `true` when the analysis succeeds. If the consumingUse cannot be used to end the lifetime, then that would be a failure of the analysis. It's nice to show an example of why that may be (use inside a loop), but I don't understand why "control equivalence" of the use would be necessary.